### ISSUE TITLE
Need remote + package version dependency

### ISSUE THREAD

**Issue by hadley** [gaborcsardi/crayon#47]
```R
crayon::blue("asdf")
#> Error: 'hasColorConsole' is not an exported object from 'namespace:rstudioapi' 
```

**Comment by gaborcsardi** [comment]
Indeed.

**Comment by gaborcsardi** [comment]
Fixed in d50dcd70e31812fd2025e2330c3e058d0ad7bebb

I cannot add a version dependency, because the GH version of rstudioapi has the same version number as the cran version.
